prow/gerrit/client/gerrit.go

```diff
@@ -93,7 +93,13 @@ func auth(c *Client) {
 	logrus.Info("Starting auth loop...")
 	for {
 		// TODO(fejta): migrate this to the grandmatriarch
-		cmd := exec.Command("python", "./git-cookie-authdaemon")
+
+		// look for authdaemon under root dir
+		if _, err := os.Stat("/git-cookie-authdaemon"); os.IsNotExist(err) {
```

Comment:
hmm can we not put this in $PATH with executable bit and a shebang?

Reply:
🙃 yeah

Reply:
I think it does

Reply:
this currently depends on it being at `/` we could instead just call `git-cookie-authdaemon` and have it somewhere in `$PATH` (with the shebang otherwise you have to do `python /path/to/authdaemon`)

Reply:
```
# stat git-cookie-authdaemon
  File: ‘git-cookie-authdaemon’
  Size: 5475      	Blocks: 16         IO Block: 4096   regular file
Device: 801h/2049d	Inode: 3010793     Links: 1
Access: (0555/-r-xr-xr-x)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 1970-01-01 00:00:00.000000000 +0000
Modify: 1970-01-01 00:00:00.000000000 +0000
Change: 2018-09-15 22:22:24.479682535 +0000
 Birth: -
```

dropped python

Reply:
should be able to drop the `/` too :-)

Reply:
where are we actually adding this to the image?

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/cmd/gerrit/BUILD.bazel#L12

I don't think I had any option, maybe the world has changed? did we bump rules_go? I can check again.

Reply:
we add this with a `container_image` base image,  https://github.com/bazelbuild/rules_docker#container_image-1

doesn't seem to be a great way to control path (you can control file placement, but adding to an env doesn't looks doable unless you already know the value) /shrug

Reply:
I bet @fejta built it with https://github.com/kubernetes/test-infra/blob/master/prow/prow.bzl which... unfortunately won't work

/shrug

once we switched to grandmatriarch we can drop all these goodies